src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -161,6 +161,17 @@ Network port to use when connecting to an Android device with [ADB (Android Devi
 
 Environment variable: `$WEB_EXT_ADB_PORT`
 
+#### \--adb-remove-old-artifacts
+
+`web-ext` does remove automatically all the temporary files that are being pushed to the target adb device when it does exit. This may fail, for example when the device is disconnected before 
+`web-ext run` exited.
+
+Starting from v5.0.0, `web-ext run` will automatically detect and warn the user if old artifacts have been found on the adb device, but it does not remove them automatically by default.
+
+This flag forces web-ext to remove automatically all the old artifacts discovered.
```

Comment:
`remove automatically` -> `automatically remove`

`all the old artifacts discovered.` -> `these discovered artifacts.`